eox_core/cms/views.py

```diff
@@ -1,19 +1,154 @@
+    """
+
+    json_content = get_json_from_body_request(request)
+    if not json_content:
+        data = {
+            'message': 'Invalid request JSON',
+            'status': 'Failed'
+        }
+        return JsonResponse(data, status=status.HTTP_400_BAD_REQUEST)
+
+    org = json_content.get('org', '')
+    user_email = json_content.get('user')
+    courses_iter, in_process_course_actions = get_courses_accessible_to_user(request, org)
+    active_courses, archived_courses = get_process_courses_list(courses_iter, in_process_course_actions)
+
+    if not org:
+        data = {
+            'message': 'Incorrect organization name.',
+            'status': 'Failed'
+        }
+        return JsonResponse(data, status=status.HTTP_400_BAD_REQUEST)
+    elif not active_courses:
+        data = {
+            'message':
+                'The organization {} does not exists, or has not active courses.'.format(org),
+            'status': 'Failed'
+        }
+        return JsonResponse(data, status=status.HTTP_404_NOT_FOUND)
+
+    data = {
+        'failed_tasks': [],
+        'complete_tasks': [],
+        'message': 'Operation complete.',
+        'status': 'Success'
+    }
+
+    # Loop over each organization course.
+    for course in active_courses:
+        plain_course_key = course.get('course_key')
+        course_key = CourseKey.from_string(plain_course_key)
+        course_team_response = get_course_team_user(request, course_key, user_email)
+        per_course_data = {}
+
+        if course_team_response.status_code != 204:
+            error_messages = json.loads(course_team_response.content)
+            per_course_data = {
+                'message': error_messages.get('error'),
+                'course': plain_course_key
+            }
+            data.get('failed_tasks').append(per_course_data)
+        else:
+            per_course_data = {
+                'message': 'success',
+                'course': plain_course_key
+            }
+            data.get('complete_tasks').append(per_course_data)
+
+    if data['failed_tasks']:
+        data['message'] = 'Some issues were found.'
+        data['status'] = 'No complete'
+    return JsonResponse(data)
+
+
+def get_all_orgs():
     """
-    Renders simple React js view.
+    Return organization list.
     """
+    organizations_raw = get_all_orgs_helper()
+    organization_list = list(organizations_raw)
+    # Add a empty value to be used inside of select html tag.
+    organization_list.insert(0, ' ')
 
-    return render_to_response(u'management.html', {'user': request.user})
+    return organization_list
+
+
+def get_json_from_body_request(request):
+    """
+    Function to convert request.body into a json format.
+    """
+    json_content = {}
+    if "application/json" in request.META.get('CONTENT_TYPE', '') and request.body:
+        try:
+            json_content = json.loads(request.body)
+        except ValueError:
+            return json_content
```

Comment:
Why return  `json_content` even if there is a `ValueError`?

Reply:
Because returns an empty dict. @diegomillan

Reply:
ok